### ISSUE TITLE
The prediction results using the model appear abnormal

### ISSUE THREAD

**Issue by DokiDokiYuyuko** [MIAGroupUT/carotid-segmentation#2]
hello.
I used the pre-training model you provided to predict the test set. I found that some parameters in the header information of the mask image were inconsistent with those of the dicom file. After I unified the two parameters, I put the modified predicted mask image and dicom file into 3d slicer for observation. It was found that the segmentation result could not be aligned with the GT(not labels after convert command), which seemed to be a centrosymmetric relationship.
For ease of viewing, I drew an outline of a slice.Red and blue are prediction results and yellow are labels. The format is converted tsv format.
![362](https://github.com/MIAGroupUT/carotid-segmentation/assets/164839734/b8599642-710f-436a-b853-44265f442e92)
